assets/javascript/app.js

```diff
@@ -360,6 +368,14 @@ window.Handlebars.registerHelper('math', function (lvalue, operator, rvalue, opt
 })
 
 document.addEventListener('DOMContentLoaded', function () {
-  window.ScreensApp.applyData(window.data)
-  window.ScreensApp.registerHandlers(true)
+  window.fetch('/save-note', {
+    method: 'POST'
+  }).then(function () {
+    window.data.noteEditingEnabled = true
```

Comment:
The data shouldn't care about the UI.

If the edit buttons are `display: none` on page load, then in the `registerHandlers` function they could be displayed if the endpoint ping is successful.